src/commands/aem/rde/inspect/inventory.js

```diff
@@ -48,7 +49,7 @@ class InventoryCommand extends BaseCommand {
         if (response.status === 200) {
           const inventory = await response.json();
           if (flags.json) {
-            this.doLog(JSON.stringify(inventory, null, 2), true);
+            result.items = inventory;
           } else {
             this.logInTableFormat([inventory]);
           }
```

Comment:
Does it hurt to always assign `result.items`? And does it hurt to always call `this.logInTableFormat([inventory]);`?

Otherwise, I would drop the if statement.